src/main/java/com/openshift/jenkins/plugins/pipeline/MessageConstants.java

```diff
@@ -0,0 +1,136 @@
+public static final String EXIT_SCALING_NOOP = "\n\nExiting \"" + OpenShiftScaler.DISPLAY_NAME + "\" successfully; no deployments for \"%s\" were found, so a replica count of \"0\" already exists.";
+public static final String EXIT_SCALING_BAD = "\n\nExiting \"" + OpenShiftScaler.DISPLAY_NAME + "\" unsuccessfully; the call to \"%s\" failed.";
+public static final String EXIT_SCALING_TIMED_OUT = "\n\nExiting \"" + OpenShiftScaler.DISPLAY_NAME + "\" unsuccessfully; the deployment \"%s\" did not reach \"%s\" replica(s) in time.";
+public static final String EXIT_SCALING_GOOD = "\n\nExiting \"" + OpenShiftScaler.DISPLAY_NAME + "\" successfully for deployment \"%s\".";
+public static final String EXIT_SCALING_GOOD_REPLICAS_GOOD = "\n\nExiting \"" + OpenShiftScaler.DISPLAY_NAME + "\" successfully, where the deployment \"%s\" reached \"%s\" replica(s).";
+
+/*
+ * These messages are for the "OpenShift ImageStream" source code management plugin
+ * Note, there aren't clear cut start and exit points with this one; the SCM has multiple 
+ * entry points based on how Jenkins decides to call it
+ */
+public static final String SCM_CALC = "\n\nThe \"%s\" SCM will return the last revision state stored in Jenkins for the image stream \"%s\" and tag \"%s\" from the project \"%s\".";
+public static final String SCM_LAST_REV = "  Last revision:  [%s]";
+public static final String SCM_NO_REV = "  No revision state has been retrieved and stored yet.";
+public static final String SCM_COMP = "\n\nThe \"%s\" SCM is pulling the lastest revision state from OpenShift for the image stream \"%s\" and tag \"%s\" from the project \"%s\" and storing in Jenkins.";
+public static final String SCM_CHANGE = "\n\n A revision change was found this polling cycle.";
+public static final String SCM_NO_CHANGE = "\n\n No revision change found this polling cycle.";
+
+/*
+ * These messages are for the "Tag OpenShift Image" jenkins build step implemented by OpenShiftImageTagger
+ */
+public static final String START_TAG = "\n\nStarting \"%s\" with the source [image stream:tag] \"%s:%s\" and destination [image stream:tag] \"%s:%s\" from the project \"%s\".";
+public static final String EXIT_OK = "\n\nExiting \"%s\" successfully.";
+
+/*
+ * These messages are for the "Verify OpenShift Service" jenkins build step implemented by OpenShiftServiceVerifier
+ */
+public static final String START_SERVICE_VERIFY = "\n\nStarting \"%s\" for the service \"%s\" from the project \"%s\".";
+public static final String SERVICE_CONNECTING = "  Attempting to connect to \"%s\" ...";
+public static final String EXIT_SERVICE_VERIFY_GOOD = "\n\nExiting \"%s\" successfully; a connection to \"%s\" was made.";
+public static final String EXIT_SERVICE_VERIFY_BAD = "\n\nExiting \"%s\" unsuccessfully; a connection to \"%s\" could not be made.";
+public static final String SOCKET_TO = " a socket level communication TIMEOUT to \"%s\" occurred.";
```

Comment:
Ah you meant the constant name - I thought you meant the message.  Will
switch.

On Monday, March 7, 2016, Ben Parees <notifications@github.com> wrote:

> In
> src/main/java/com/openshift/jenkins/plugins/pipeline/MessageConstants.java
> <https://github.com/openshift/jenkins-plugin/pull/23#discussion_r55233664>
> :
>
> > +public static final String SCM_NO_CHANGE = "\n\n No revision change found this polling cycle.";
> > +
> > +/*
> > + * These messages are for the "Tag OpenShift Image" jenkins build step implemented by OpenShiftImageTagger
> > + */
> > +public static final String START_TAG = "\n\nStarting \"%s\" with the source [image stream:tag] \"%s:%s\" and destination [image stream:tag] \"%s:%s\" from the project \"%s\".";
> > +public static final String EXIT_OK = "\n\nExiting \"%s\" successfully.";
> > +
> > +/*
> > + * These messages are for the "Verify OpenShift Service" jenkins build step implemented by OpenShiftServiceVerifier
> > + */
> > +public static final String START_SERVICE_VERIFY = "\n\nStarting \"%s\" for the service \"%s\" from the project \"%s\".";
> > +public static final String SERVICE_CONNECTING = "  Attempting to connect to \"%s\" ...";
> > +public static final String EXIT_SERVICE_VERIFY_GOOD = "\n\nExiting \"%s\" successfully; a connection to \"%s\" was made.";
> > +public static final String EXIT_SERVICE_VERIFY_BAD = "\n\nExiting \"%s\" unsuccessfully; a connection to \"%s\" could not be made.";
> > +public static final String SOCKET_TO = " a socket level communication TIMEOUT to \"%s\" occurred.";
>
> still says TO
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/jenkins-plugin/pull/23/files#r55233664>.
>